### PR TITLE
Benchmarking per channel quantization

### PR DESCRIPTION
**Pull request by z-a-f**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#29627 Benchmarking per channel quantization**

Differential Revision: [D18443929](https://our.internmc.facebook.com/intern/diff/D18443929)

### CODE REVIEW

File: benchmarks/operator_benchmark/pt/quantization_test.py [hunk 1]
```diff
@@ -55,6 +55,42 @@ def forward(self):
...
+
+        scales = torch.tensor([1.0] * C)
+        zero_points = torch.tensor([0] * C)
+        dtype = dtype
+
+        self.kwargs = {
+            'scales': scales,
+            'zero_points': zero_points,
+            'dtype': dtype,
+            'axis': 0
```
Comment (raghuramank100) [inline benchmarks/operator_benchmark/pt/quantization_test.py]: Suggest adding one non-zero axis argument as the implementation could be slower when the axis is not zero

**Review by raghuramank100 — APPROVED** [review]
